### PR TITLE
Micah dev

### PR DESCRIPTION
**Pull request by Ixw123**
Envelop and Center are properly compiling at least BBox still needs some works in an outlined area, please let me know what you think needs to be done in BBox for that one function because im not quite sure. I will port the last couple remaining previously assigned files today to the best of my ability and have another PR for you then.

### CODE REVIEW

**Comment by hudsonb** [comment]
The BBox class is immutable, you aren't supposed to be able to modify those
values post construction. Create vars to track south/West/north/East then
simply construct the BBox at the end with those bars.

**Comment by Ixw123** [comment]
Oh okay I think I know how I need to change it then. I'll make the changes
after i get some shuteye. So the bbox extension functions essentially would
fill in the bbox value of and only if it is null right?

File: src/main/kotlin/geojsonkt/BBox.kt
Comment (hudsonb) [inline src/main/kotlin/geojsonkt/BBox.kt:10]: Remove commented out code

File: src/main/kotlin/geojsonkt/BBox.kt
Comment (hudsonb) [inline src/main/kotlin/geojsonkt/BBox.kt:65]: Add support for bbox on MultiPoint. Even if turf.js doesn't support it, it seems easy enough to implement.

File: src/main/kotlin/geojsonkt/BBox.kt
Comment (hudsonb) [inline src/main/kotlin/geojsonkt/BBox.kt:77]: Remove comment

File: src/main/kotlin/turfkt/Midpoint.kt
Comment (hudsonb) [inline src/main/kotlin/turfkt/Midpoint.kt:9]: Remove `units`. It should be an optional parameter in `destination`, so you don't need to provide since it.

File: src/main/kotlin/turfkt/Midpoint.kt
Comment (hudsonb) [inline src/main/kotlin/turfkt/Midpoint.kt:28]: 1. Remove unneeded `this` references throughout the file.

2. `Point.midpoint` should just defer to `Position.midpoint`:
```
fun Point.midpoint(to: Position): Position =
    coordinate.midpoint(to)  
```

Same for `Feature<Point>.midpoint`. We don't want to repeat the logic for calculating the midpoint multiple times, it becomes harder to maintain.

File: src/main/kotlin/turfkt/CenterOfMass.kt
Comment (hudsonb) [inline src/main/kotlin/turfkt/CenterOfMass.kt:1]: This file does not compile. You'll have to get it at least compiling before this PR can be merged.

**Review by hudsonb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
